os/drivers/testcase/testcase_drv.c

```diff
@@ -72,8 +78,95 @@ static int testdrv_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	 *   ioctl argument:  An integer value indicating the particular test to be run
 	 */
 
-	case TESTIOC_DRIVER_ANALOG: {
+	case TESTIOC_ANALOG: {
+
+	}
+	break;
+
+	case TESTIOC_CLOCK_ABSTIME2TICKS: {
+		base_time.tv_sec = (time_t)arg;
+		base_time.tv_nsec = 0;
+		if (clock_abstime2ticks(CLOCK_REALTIME, &base_time, &ret) != OK) {
+			ret = ERROR;
+		}
+	}
+	break;
+
+	case TESTIOC_GET_SELF_PID: {
+		tcb =  sched_self();
+		if (tcb == NULL) {
+			ret = ERROR;
+			break;
+		}
+		ret = tcb->pid;
+	}
+	break;
+
+	case TESTIOC_GET_SIG_FINDACTION_ADD: {
+		ret = (int)sig_findaction(sched_self(), 2);
```

Comment:
2 is the signal number? please change the define value like SIGINT.

Reply:
okay. i will fix.